lib/ui/platform_dispatcher.dart

```diff
@@ -801,11 +801,36 @@ class PlatformDispatcher {
   ///
   ///  * [SchedulerBinding], the Flutter framework class which manages the
   ///    scheduling of frames.
+  ///  * [forceSyncFrame], a similar method that is used in rare cases that
+  ///    a frame must be rendered immediately.
   void scheduleFrame() => _scheduleFrame();
 
   @Native<Void Function()>(symbol: 'PlatformConfigurationNativeApi::ScheduleFrame')
   external static void _scheduleFrame();
 
+  /// Immediately render a frame by invoking the [onBeginFrame] and
+  /// [onDrawFrame] callbacks synchronously.
```

Comment:
We should not document that this happens synchronously. We should just document that this allows initiation of a warmup frame independent of whether the engine has processed a vsync signal yet, and that it is intended for use only for the purpose of rendering a warmup frame at program startup.